package.json

```diff
@@ -68,5 +68,8 @@
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"
+  },
```

Comment:
I don't think we should add the edition in the package.json until we update the version of ember-cli (we're currently on ember-cli 3.3)

Reply:
FYI I've started trying to upgrade ember-cli in https://github.com/lytics/ember-data-model-fragments/pull/385

Reply:
@Gorzas if you pull the latest changes from master you should be able to remove these changes now